Decred Wallet/Features/Receive/ReceiveViewController.swift

```diff
@@ -0,0 +1,221 @@
+//  ReceiveViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+import UIKit
+import Dcrlibwallet
+
+class ReceiveViewController: UIViewController, UIDocumentInteractionControllerDelegate {
+    @IBOutlet weak var menuBtn: UIButton!
+    @IBOutlet private var imgWalletAddrQRCode: UIImageView!
+
+    @IBOutlet weak var walletAddressLabel: UILabel!
+    @IBOutlet weak var containerRoundedView: RoundedView!
+
+    @IBOutlet weak var selectedAccountView: UIView!
+    @IBOutlet weak var accountNameLabel: UILabel!
+    @IBOutlet weak var walletNameLabel: UILabel!
+    @IBOutlet weak var totalAccountBalanceLabel: UILabel!
+
+    private lazy var syncInProgressLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .darkGray
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.text = LocalizedStrings.secureMenuSyncInfo
+        return label
+    }()
+
+    var tapGesture = UITapGestureRecognizer()
+    var oldAddress = ""
+
+    var selectedWallet: DcrlibwalletWallet?
+    var selectedAccount: DcrlibwalletAccount?
+
+    override func loadView() {
+        super.loadView()
+        view.addSubview(syncInProgressLabel)
```

Comment:
If we're adding this sync progress label with the other views when the page's view is being loaded, then let's just add it to the storyboard. That would mean we won't also need the extra lines of code in the `setupSyncInProgressLabelConstraints` function.